persistence/secrets.go

```diff
@@ -0,0 +1,112 @@
+package persistence
```

Comment:
The key is now service org/ service url/ secret name. Then that will return a map of service versions to the secrets. This is because the secrets are coming in with version ranges and need to be findable with individual versions